src/ffi/wrapper.hpp

```diff
@@ -0,0 +1,44 @@
+// header wrapper to feed into bindgen
+
+#include "msquic.h"
+
+// bindgen or clang has problem with c marcro functions
```

Comment:
But this requires people to know to update this file if a new QUIC_STATUS_* is added. Any way to automate validation for that?